README.md

```diff
@@ -12,6 +12,12 @@
 
 ### Save in ArchivesSpace
 
+The RAC_ASpace library improves upon the ArchivesSnake saving functionality by gerneralizing processes and making it easier to save new JSON data and push that back up to ArchivesSpace.
+
+We plan to do that by:
+  * Generalizing the transformation from a JSONModel object fetched by ASNake to editable JSON
```

Comment:
Is there anything more involved than just calling the `json()` method on the `JSONModel` object?

Reply:
No, I don't really think there is, so I'm not sure how much work there is to do here. I think there is maybe potential to tie calling that method into some other actions, but that might crossover with fetching.